### PR TITLE
Increase timeout, and call setIsLoading in catch

### LINKED ISSUE #151: wrapper timeout option needs to be incremented
## Expected Behavior
Pokemon should be fetched as user scrolls.

## Actual/Current Behavior
Sometimes timesout after 5000ms error occurs and break app.

### If you are not a member of this project, _skip this step_

_How do the users test this change?_

  1. Navigate to home screen
  2. Scroll
  3. Observe

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Increased timeout property in `customOptions` of wrapper.
2. Invoke `setIsLoading` in `catch` of `getMorePokemons`.

## Purpose
Prevent errors and spinner from staying up once user scrolls past 898 pokemons(due to API data gap).

## Testing Steps
Scroll all the way down on home page.

Closes #151

### CODE REVIEW

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const PokeApp = () => {
         limit: queryParams.limit,
       });
     } catch (error) {
+      setIsLoading(false);
```
Comment (dannysanchez559) [inline src/components/PokeApp.js:60]: nice handling

**Review by dannysanchez559 — APPROVED** [review]
short and sweet

File: src/util/fetchPokemonData.js [hunk 2]
```diff
@@ -3,14 +3,13 @@ const Pokedex = require("pokeapi-js-wrapper");
 
 const customOptions = {
   protocol: "https",
-  // hostName
   versionPath: "/api/v2/",
   cache: true,
-  timeout: 5 * 1000, // 5s
+  timeout: 10 * 1000, // 5s
```
Comment (JorgeLVilla) [inline src/util/fetchPokemonData.js:8]: the details!

**Review by JorgeLVilla — APPROVED** [review]
Looks great!

File: src/util/fetchPokemonData.js (hunk 2 above)
Comment (BlackCubes) [inline src/util/fetchPokemonData.js:8]: I like dis

**Review by BlackCubes — APPROVED** [review]
Nice catch on the timeout

**Pull request merged**